common/src/main/java/com/datastax/oss/simulacron/common/cluster/QueryLog.java

```diff
@@ -97,6 +98,11 @@ public QueryLog(
           }
         }
       }
+    } else if (frame.message instanceof Prepare) {
+      Prepare prepare = (Prepare) frame.message;
+      this.consistency = ConsistencyLevel.ANY;
+      this.serialConsistency = ConsistencyLevel.SERIAL;
+      this.query = String.format("PREPARE (%s)", prepare.cqlQuery);
```

Comment:
:+1:, although what do you think about adding a new field to `QueryLog` called `type` that simply has the type of message in all caps (`frame.message.getClass().getSimpleName().toUpperCase();`).  Then the query field can just contain `prepare.cqlQuery`.  That would provide a more uniform way to test for the message type (i.e. is it a prepare message) and then subsequently the associated query.

Reply:
Agree